script/upgrades/dependencies.json

```diff
@@ -18,7 +18,10 @@
     "MentoLiquiditySupport": "0xA74Ac93de1A209957E62391B01E09161277a9ffC",
     "WatchdogMultisig": "0xE6951C4176aaB41097C6f5fE11e9c515B7108acd",
     "FractalSigner": "0xacD08d6714ADba531beFF582e6FD5DA1AFD6bc65",
-    "MentoFoundationMultisig": "0x3468D23A0B1aB3Ab9A537813166A8f7ff1947014"
+    "MentoFoundationMultisig": "0x3468D23A0B1aB3Ab9A537813166A8f7ff1947014",
+    "Chainlink.CELOUSD": "0x0568fD19986748cEfF3301e55c0eb1E729E0Ab7e",
+    "Chainlink.USDTUSD": "0x5e37AF40A7A344ec9b03CCD34a250F3dA9a20B02",
+    "Chainlink.PHPUSD": "0x4ce8e628Bb82Ea5271908816a6C580A71233a66c"
```

Comment:
are we missing `Chainlink.CELOPHP` here or left out on purpose?

Reply:
CELOPHP doesn't exist on Chainlink, these are the addresses of Chainlink aggregators, and we're combining CELOUSD and PHPUSD to report CELOPHP